### ISSUE TITLE
cron not working with commas

### ISSUE THREAD

**Issue by Loader009** [anatol/pacoloco#36]
I defined a few specific hours for downloading automatically.
This doesn't seem to work, as my journal told me `journalctl -t pacoloco`.

In pacoloco.yml:
```yaml
prefetch:
  cron: 0 30 1,9,16 * * * *
  ttl_unaccessed_in_days: 30
  ttl_unupdated_in_days: 300
```

This should mean: on 30 minutes at 1 am, 9 am and 4 pm, everyday.
Have I done something wrong or is the cron implementation wrong?

**Comment by anatol** [comment]
cc @Focshole 

The cron implementation comes from this library https://github.com/gorhill/cronexpr And per its description it supports comma delimiter https://github.com/gorhill/cronexpr#comma---

```
Commas are used to separate items of a list. For example, using MON,WED,FRI in the 5th field (day of week) means Mondays, Wednesdays and Fridays.
```

There are also tests that show example of comma lists https://github.com/gorhill/cronexpr/blob/master/cronexpr_test.go#L134

So my guess your example should work unless there is some corner-case in `cronexpr` library.

**Comment by Focshole** [comment]
After testing with 
`main.go`
```
package main

import (
	"fmt"
	"time"

	"github.com/gorhill/cronexpr"
)

func main() {
	now := time.Now()
	timeToCheck := now.Add(time.Hour * 16) // change this to get it working
	fmt.Printf("time to check: \t\t\t%v\n", timeToCheck)
	fmt.Printf("next tick from time to check: \t%v\n", (cronexpr.MustParse("0 30 1,9,16 * * * *").Next(timeToCheck)))
}
```
Which can be run with
```
go get github.com/gorhill/cronexpr
go run ./main.go
```
I do get proper results, the issue is not in gorhill library.
I had checked the pacoloco code too, I installed it with the `pacoloco-git` AUR and I couldn't replicate the issue at all.

Then I tried with the `pacoloco` package and I could replicate the issue. Is the `pacoloco` official arch linux package out of date?

**Comment by Focshole** [comment]
Ok, yes, it is definitely out of date (last build was on april 2021, the prefetching code had been merged around august). 
As a temporary fix, use the `pacoloco-git` AUR package. I'll flag it as out-of-date.

**Comment by anatol** [comment]
> official arch linux package out of date?

It is at version `1.0` from April. It might be a good idea to make a new release and update `pacoloco` package.

@Loader009 could you please try `pacoloco-git` package from AUR to see if you still have this problem.

**Comment by Focshole** [comment]
Clearly on April there was no prefetching, hence the prefetching isn't working at all nor being parsed. I think it would be a good idea to make a new release.

**Comment by Loader009** [comment]
I'm seeing this now:
`pacoloco[2969405]: prefetch.go:34: The prefetching routine will be run on 2021-11-18 03:00:00.000174124 +0100 CET m=+25279.173439789`
So I guess the official package just needs an update.

Thanks for pointing it out!

**Comment by anatol** [comment]
Thanks for confirming.

Also I just released pacoloco 1.1 and pushed it to [community-testing] repo. Please try it and let me know if you see any issues.
